manifests/plugin/file.pp

```diff
@@ -1,5 +1,17 @@
 # @summary Pulp File plugin
-class pulpcore::plugin::file {
+# @param redirect_legacy
+#   Whether to redirect the legacy (Pulp 2) URLs to the content server
+class pulpcore::plugin::file (
+  Boolean $redirect_legacy = false,
```

Comment:
I lean towards being more explicit here since legacy can be a generic term. Something like `use_pulp_2_content_route`

Reply:
Good point and I like your suggestion.